uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/annual-entitlement-change-log.component.js

```diff
@@ -77,10 +76,12 @@ define([
      * that has comments.
      */
     function highlightEntitlementWithComments () {
+      var changeLogRow, entitlementComments, validEntitlementComments;
+
       for (var i = vm.changeLogRows.length - 1; i >= 0; i--) {
-        var changeLogRow = vm.changeLogRows[i];
-        var entitlementComments = _.pluck(changeLogRow.entitlements, 'comment');
-        var validEntitlementComments = _.compact(entitlementComments).length;
+        changeLogRow = vm.changeLogRows[i];
+        entitlementComments = _.pluck(changeLogRow.entitlements, 'comment');
+        validEntitlementComments = _.compact(entitlementComments).length;
```

Comment:
Not sure if the name `validEntitlementComments` is appropriate as its actually holds the **number** of entitlement comments.